imio/dms/mail/tests/test_utils.py

```
@@ -1026,3 +1027,107 @@ def test_eml_preview(self):
         with open(os.path.join(PREVIEW_DIR, "previsualisation_eml_normal.jpg"), 'rb') as f1, \
                 open(blobs[0]._p_blob_uncommitted, 'rb') as f2:
             self.assertEqual(f1.read(), f2.read())
+
+    def test_update_solr_config(self):
+        get_rec = "imio.dms.mail.utils.api.portal.get_registry_record"
+        set_rec = "imio.dms.mail.utils.api.portal.set_registry_record"
+
+        def make_patches(registry):
+            """Return get/set patches that read from and write to registry."""
+            get_se = lambda k, default=None: registry.get(k, default)
```

Ruff (0.15.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `get_se` as a `def`

(E731)